roboteam_ai/src/io/IOManager.cpp

```diff
@@ -107,6 +107,15 @@ void IOManager::handleGeometryData(const roboteam_msgs::GeometryDataConstPtr &ge
 void IOManager::handleRobotFeedback(const roboteam_msgs::RobotFeedbackConstPtr &robotfeedback) {
```

Comment:
Is this a special type made for constPtrs from Ros or are you making a const ptr of a const ptr? No harm in making it explicit in c++ but why not just const ptr of RobotFeedback?

Reply:
Its autogenerated from ROS

Reply:
I am just interested in what the difference between the following two would be:
const roboteam_msgs::RobotFeedbackConstPtr &robotfeedback
const roboteam_msgs::RobotFeedback &robotfeedback